### PR TITLE
Rename "magpie" to "beneficiary"

### PR DESCRIPTION
**Pull request by eth-r**
This should cover all appearances.

### CODE REVIEW

**Comment by nkuba** [comment]
Please remember about updating references to magpie in `keep-ecdsa` as well.

**Comment by eth-r** [comment]
Done: https://github.com/keep-network/keep-ecdsa/pull/431

File: solidity/scripts/manage-stake.js [hunk 1]
```diff
@@ -4,7 +4,7 @@
   To stake KEEP tokens, use 'stake' command and provide three parameters:
   - operator address
   - amount of KEEP to stake
-  - KEEP owner address (we assume it's also magpie/beneficiary/authorizer address)
+  - KEEP owner address (we assume it's also beneficiary/beneficiary/authorizer address)
```
Comment (pdyraga) [inline solidity/scripts/manage-stake.js:7]: `beneficiary/beneficiary/authorizer` -> `beneficiary/authorizer`

File: solidity/scripts/rewards.js [hunk 2]
```diff
@@ -2,7 +2,7 @@
  * Important: the KeepRandomBeaconOperatorRewardsStub contract should be deployed to the network!
  * 
  * This script:
- *  - delegates stake to operators. For each operator uses the same beneficiary(magpie) address,
+ *  - delegates stake to operators. For each operator uses the same beneficiary(beneficiary) address,
```
Comment (pdyraga) [inline solidity/scripts/rewards.js:5]: `beneficiary(beneficiary)` -> `beneficiary`

**Comment by dimpar** [comment]
Tested:
- run genesis - looks fine
- request a relay entry - looks fine

**Comment by dimpar** [comment]
Dashboard looks good too. Tested staking / grant creation / undelegation.

**Pull request merged**